woocommerce/woocommerce-ios
# Analytics Hub: Integrate Custom Range Selector UI

Closes: #8200

# Why

This PR connects the custom range selector and passes the selected ranges down to the view model.

# How

- Update Range Card View to present the custom range selector when the custom range type is selected.

- Makes use of an internal binding in the range card view to track when the custom range type is pressed in order to not pass that immediately to the view model but to wait for the selected range.

# Demo


https://user-images.githubusercontent.com/562080/206313407-3a8789e7-fefc-4f05-a656-0e357053cff9.mov


# Testing

- Open the analytics hub
- Select a custom time range
- See that the custom range is loaded.

Note: There are two couple of known issues being tracked here
- https://github.com/woocommerce/woocommerce-ios/issues/8343
- https://github.com/woocommerce/woocommerce-ios/issues/8344

---
